cluster-cm/src/test/java/com/sequenceiq/cloudbreak/cm/ClouderaManagerSetupServiceTest.java

```diff
@@ -1004,6 +1009,113 @@ public void testInstallClusterWhenEverythingWorksFineShouldPollTheInsallProgress
         verify(clusterCommandRepository, times(1)).save(any(ClusterCommand.class));
     }
 
+    @Test
+    void getCMHosFromMGMTWithEnterpiseDL() throws ApiException, IOException {
+        ClouderaManagerSetupService spy = spy(underTest);
+
+        ApiHostList apiHostList = getApiHostList();
+
+        var template = FileReaderUtils.readFileFromPath(Path.of("../core/src/main/resources/defaults/blueprints/7.2.17/cdp-sdx-enterprise.bp"));
```

Comment:
could you use the actual types instead of `var`?

Reply:
addressed,

Reply:
I still see var everywhere